problems/countNumbers.js

```diff
@@ -9,8 +9,12 @@
  * returns { 1:3, 2:2, 3:1, 4:1 }
  */
 
-function countNumbers() {
- 
+function countNumbers(arr) {
+    let count = {}
```

Comment:
Very clean. Can use const for count as well because we never do reasiggnment of variable.